api.json

```diff
@@ -0,0 +1,761 @@
+      },
+      "/:id/organisations": {
+        "GET": {
+          "description": "",
+          "auth": true,
+          "scope": [],
+          "query": {
+            "fields": ["String"]
+          },
+          "response": {
+            "totalOrganisations": "Number",
+            "organisationsList":[{
+              "id": "Number",
+              "name": "String",
+              "logo_url": "String",
+              "address1": "String",
+              "address2": "String",
+              "townCity": "String",
+              "postCode": "String",
+              "coordinates": "String",
+              "region": "String",
+              "sector": "String",
+              "registeredAt": "String",
+              "lastActiveAt": "String",
+              "modifiedAt": "String",
+              "turnoverBand": "String",
+              "360GivingId": "String",
+              "fundingProgramme": "String",
+              "fundingBody": "String",
+              "parentOrganisation": "String",
+              "childOrganisations": ["String"],
+              "subscription": "String",
+              "trainingSessions": [{
+                "trainingSessionAttendees": ["Number"],
+                "trainingSessionNotes": "String",
+                "eventDate": "String"
+              }]
+            }]
+          }
+        }
+      }
+    },
+    "/regions": {
+      "description": "",
+      "/:id/organisations": {
+        "GET": {
+          "description": "",
+          "auth": true,
+          "scope": [],
+          "query": {
+            "fields": ["String"]
+          },
+          "response": {
+            "totalOrganisations": "Number",
+            "organisationsList":[{
+              "id": "Number",
+              "name": "String",
+              "logo_url": "String",
+              "address1": "String",
+              "address2": "String",
+              "townCity": "String",
+              "postCode": "String",
+              "coordinates": "String",
+              "region": "String",
+              "sector": "String",
+              "registeredAt": "String",
+              "lastActiveAt": "String",
+              "modifiedAt": "String",
+              "turnoverBand": "String",
+              "360GivingId": "String",
+              "fundingProgramme": "String",
+              "fundingBody": "String",
+              "parentOrganisation": "String",
+              "childOrganisations": ["String"],
+              "subscription": "String",
+              "trainingSessions": [{
+                "trainingSessionAttendees": ["Number"],
+                "trainingSessionNotes": "String",
+                "eventDate": "String"
+              }]
+            }]
+          }
+        }
+      }
+    },
+    "/organisations": {
+      "description": "",
+      "/": {
+        "GET": {
+          "description": "",
+          "auth": {},
+          "scope": {},
+          "query": {},
+          "response": {}
+        }
+      },
+      "/:id||me": {
```

Comment:
I'm not sure this makes sense. `me` is really about the user, rather than the organisation or any other entity. 

Also remember that in the future, users (volunteers/visitors) may be related to several different organisations, so which would we choose?

To begin with we can have a constraint implemented in application logic that restricts 1 user to 1 organisation only (since it corresponds to assumptions made so far and application UI design so far), and then remove it down the line.

I think `/users/me/organisations` is better, and to start with it can just always return an array of one organisation.

What do you think?